build-image/Dockerfile

```diff
@@ -30,15 +30,14 @@ RUN GOARCH=$(go env GOARCH) && \
 
 RUN curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b /usr/bin v1.27.0
 
-RUN GO111MODULE=on go get \
-	github.com/client9/misspell/cmd/misspell@v0.3.4 \
-	github.com/golang/protobuf/protoc-gen-go@v1.3.1 \
-	github.com/gogo/protobuf/protoc-gen-gogoslick@v1.3.0 \
-	github.com/gogo/protobuf/gogoproto@v1.3.0 \
-	github.com/weaveworks/tools/cover@bdd647e92546027e12cdde3ae0714bb495e43013 \
-	github.com/fatih/faillint@v1.5.0 \
-	github.com/campoy/embedmd@v1.0.0 \
-	&& rm -rf /go/pkg /go/src /root/.cache
+RUN GO111MODULE=on go install github.com/client9/misspell/cmd/misspell@v0.3.4 &&\
```

Comment:
`go get` no longer supports installing binary. See https://go.dev/doc/go-get-install-deprecation#:~:text=Starting%20in%20Go%201.17%2C%20installing,the%20%2Dd%20flag%20were%20enabled.